src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -462,7 +462,19 @@ private void PackageOnExecuted(PackageManagerSearchElement element, PackageVersi
                     return;
                 }
 
-                var allPackageVersions = PackageManagerSearchElement.ListRequiredPackageVersions(headers, version);
+                List<Tuple<PackageHeader, PackageVersion>> allPackageVersions = null;
+                try
+                {
+                    allPackageVersions = PackageManagerSearchElement.ListRequiredPackageVersions(headers, version).ToList();
```

Comment:
We need to validate the generated list from `ListRequiredPackageVersions` -> `.Select`; 
`Select` is implemented by using deferred execution. TO validate, I called `.ToList` inside a try catch..
https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.select?view=netframework-4.7.2

Reply:
Thanks.